.github/workflows/tests.yml

```diff
@@ -49,17 +50,13 @@ jobs:
           source venv/bin/activate 
           pip freeze
 
-      - name: Run make lint
+      - name: Run make lint-check
```

Comment:
The lint-check task runs the check-only version against the source and test code. If this fails, the build breaks, as desired.